src/mpris.cpp

```diff
@@ -17,8 +17,75 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#include "mpris_p.h"
 #include "mpris.h"
 
 using namespace Amber;
 
 Mpris::Mpris() {}
+
+Mpris::LoopStatus MprisPrivate::LoopStatusStringToEnum(const QString &value, bool *ok) {
+    Mpris::LoopStatus enumVal = Mpris::LoopNone;
+    bool found = true;
+
+    if (value == QLatin1String("None")) {
+        enumVal = Mpris::LoopNone;
+    } else if (value == QLatin1String("Track")) {
+        enumVal = Mpris::LoopTrack;
+    } else if (value == QLatin1String("Playlist")) {
+        enumVal = Mpris::LoopPlaylist;
+    } else {
+        found = false;
+    }
+
+    if (ok) {
+        *ok = found;
+    }
+
+    return enumVal;
+}
+
+QString MprisPrivate::LoopStatusEnumToString(Mpris::LoopStatus value) {
+    switch (value) {
+    case Mpris::LoopNone:
+        return QLatin1String("None");
```

Comment:
Would QStringLiteral() be better here, as the value is passed in a QString.

Reply:
Thanks for the comments @inzanity. I agree, I've changed it to `QStringLiteral()`.